rtf_vis_tool/src/Components/MVOSummary.js

```diff
@@ -0,0 +1,86 @@
+/* Component to render Multiview Optimizer Metrics. Specifically rotation averaging and
+translation averaging metrics.
+
+Author: Adi Singh
+*/
+import React from "react";
+
+// Local Imports.
+import '../stylesheets/MVOSummary.css'
+
+function MVOSummary(props) {
+    /*
+    Args:
+        props.json.rotation_averaging_angle_deg.median_error (float): Median error of rotation averaging.
+        props.json.rotation_averaging_angle_deg.min_error (float): Min error of rotation averaging.
+        props.json.rotation_averaging_angle_deg.max_error (float): Max error of rotation averaging.
+        props.json.translation_averaging_distance.median_error (float): Median error of translation averaging.
+        props.json.translation_averaging_distance.min_error (float): Min error of translation averaging.
+        props.json.translation_averaging_distance.max_error (float): Max error of translation averaging.
+        props.json.translation_to_direction_angle_deg.median_error (float): Median error of translation to direction.
+        props.json.translation_to_direction_angle_deg.min_error (float): Min error of translation to direction.
+        props.json.translation_to_direction_angle_deg.max_error (float): Max error of tranlation to direction.
```

Comment:
I prefer: `Max Euclidean error on global translation for any frame after translation averaging (pre-BA)`